dev-libs/capstone/capstone-5.0.1.ebuild

```diff
@@ -62,7 +62,7 @@ src_prepare() {
 
 src_configure() {
 	local mycmakeargs=(
-		-DCAPSTONE_BUILD_TESTS="$(usex test)"
+		-DCAPSTONE_BUILD_TESTS=ON
```

Comment:
> build tests only when necessary

I assume by this that you mean the upstream build system *does* only build tests when necessary already, and thus this option doesn't need to be passed?

Does this also mean we can get rid of IUSE=test and RESTRICT=?

Reply:
Sorry, disregard this commit. This was an error on my side. For some reason I read the patch the other way round when reverting from some experiments. The former ebuild was right and I dropped the commit from the PR.